packages/codegen-ui-react/lib/react-component-render-helper.ts

```diff
@@ -146,29 +146,32 @@ export function buildFixedJsxExpression(prop: FixedStudioComponentProperty): Str
     case 'string':
       switch (type) {
         case undefined:
-          return factory.createStringLiteral(value as string);
```

Comment:
This will change the existing behavior when no type is passed. Is this intentional?

```
{
  "value": "{ foo: bar }"
}

<Label prop="{ foo: bar }" />
```

Will change to:

```
{
  "value": "{ foo: bar }"
}

<Label prop={{ foo: bar }} />
```

Reply:
The statement under case `undefined` and case `string` was exactly the same before my changes, I just re-arranged them to reduce code duplication

Reply:
Ah, I understand. You are using fall-through here 👍